utils/display.ts

```diff
@@ -12,3 +13,22 @@ export function timeAgo(time: number | Date) {
   else if (minutes < 24 * 60) return pluralize(~~(minutes / 60), "hour");
   else return pluralize(~~(minutes / (24 * 60)), "day");
 }
+
+/**
+ * Dynamically generates link styles depending on whether the given expression is truthy.
+ * This is used to visually highlight a link if it matches the current page.
+ *
+ * @example
+ * ```ts
+ * import { getLinkStyles } from "@/utils/display.ts";
+ *
+ * const activeLinkStyles = getLinkStyles(true);
+ *
+ * const inactiveLinkStyles = getLinkStyles(false);
+ * ```
+ */
+export function getLinkStyles(expr: boolean) {
```

Comment:
I'd imagine there could be more variations in the future (e.g. disabled, etc). How about making this `getLinkStyles(options: { active: boolean })` to prepare for the future extension?

Reply:
Are you referring to the [`:disabled` CSS pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:disabled)? If so, this function only takes care of the styles while the element is at its "rest state" (without any of these states active).

Reply:
Would something like `getStyles(baseStyles: string, toggledStyles: string, expr?: boolean)` do the trick?

Reply:
I've implemented my above suggestion. WDYT?